src/exporter/index.js

```diff
@@ -33,7 +33,7 @@ module.exports = (hash, ipldResolver, options) => {
       return pull.empty()
     }
     return pull(
-      ipldResolver.getStream(new CID(item.hash)),
+      ipldResolver._getStream(new CID(item.hash)),
```

Comment:
I can expose it again with the new version that also understands paths. (The internal battle between less is more or more being actually more)